docs/CHANGELOG.asciidoc

```diff
@@ -30,6 +30,223 @@ This section summarizes the changes in each release.
+Geo::
+* [Fixes #34399] [Maps] add Where clause to terms joins {pull}39593[#39593]
+* [Maps] increase document limit from 2048 to 10000 {pull}39697[#39697]
+* [Maps] remove beta labeling from application and docs {pull}39871[#39871]
+* [Fixes #31697] [Maps] Custom labels for metric aggregations {pull}39914[#39914]
+* [Maps] replace legacy map visualizations with maps in sample data dashboards {pull}35997[#35997]
+* [Fixes #34026] [Maps] make EMS tooltips configurable {pull}34325[#34325]
+* [Fixes #29005] [Maps] add multiple feature selection in tooltips {pull}39629[#39629]
+Infra Logs UI::
+* [Logs UI] Add single phrase highlighting {pull}39569[#39569]
+* [InfraUI] Improve accuracy of system network metrics {pull}39513[#39513]
+* [Infra UI] Add default metrics to metrics explorer. {pull}39787[#39787]
+* [Infra UI] Use max instead of avg for correct derivative in network metrics {pull}41009[#41009]
+* [Fixes #38167] [Logs UI] Add &quot;View in APM&quot; link to log flyout actions menu  {pull}39540[#39540]
+Machine Learning::
+* [Fixes #39240] [ML] When importing CSV through file data viz, omit empty values {pull}39524[#39524]
+* [Fixes #39242] [ML] Don't quote numbers when importing CSV in file data viz {pull}39579[#39579]
+* [ML] Adds data frame messages to data frames list {pull}39609[#39609]
+* [ML] Adds icon to data frame messages expanded row tab {pull}39817[#39817]
+* [ML] Adds Preview table tab to Data Frames list expanded row {pull}39983[#39983]
+* [ML] Data Frames list expanded row preview: Fixes transform list paging and sorting. {pull}40163[#40163]
+* [ML] Adds SIEM auditbeat and metricbeat data recognizer modules {pull}39678[#39678]
+* [Fixes #28390] [ML] [Accessibility] Add job ID to select row checkbox aria-label in Jobs List {pull}40149[#40149]
+* [ML] Data frames: Advanced editor {pull}39659[#39659]
+* [ML] Data Frames: Continuous mode support for wizard {pull}39804[#39804]
+* [ML] Data frames: Updated progress reporting. {pull}39920[#39920]
+* [ML] Data Frames: Transform list refresh button. {pull}40129[#40129]
+* [Fixes #38776] [ML] Data Frames: Adds support for transform description {pull}40153[#40153]
+Management::
+* [Fixes #13804, #17894, #17956, #17969, #17978, #18008, #18008, #18031, #18047, #18052, #18151, #18191, #18245, #18246, #18256, #18268, #18271, #18296, #18353, #18506, #18508, #18532, #18536, #18794, #22615, #29167, #33207, #33917, #34932, #39828] EUIfy Watcher {pull}35301[#35301]
+* Separate Console History panel from editor with a drop-shadow, and add a Close button {pull}39972[#39972]
+* [Fixes #39985] Accept triple quoted input in Search Profiler. {pull}39984[#39984]
+* [SR] Snapshot and Restore UI {pull}39193[#39193]
+* EUIfy Console - partially de-angularize and move custom views out of top_nav {pull}39341[#39341]
+* Console update autocomplete definitions {pull}39508[#39508]
+Monitoring::
+* Removed dead code {pull}39545[#39545]
+* Issue 5429: Added a shortcut to ML jobs {pull}39147[#39147]
+Observability::
```

Comment:
App is Uptime, so this section should be called that and we should remove [Uptime] prefixes